apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/components/spotlight/index.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * External Dependencies
+ */
+import { useRef, useMemo } from '@wordpress/element';
+import classnames from 'classnames';
+import { usePopper } from 'react-popper';
+import type { Rect, Placement } from '@popperjs/core';
+
+interface Props {
+	referenceElementSelector: string;
+}
+
+const Spotlight: React.FunctionComponent< Props > = ( { referenceElementSelector } ) => {
+	const popperElementRef = useRef( null );
+	const referenceElement = document.querySelector( referenceElementSelector );
```

Comment:
Could it be the same if we pass directly `referenceElement` from `tour-launch.js`, without using`document.querySelector` here? Or you did this way for performance reasons (passing Element as a prop)? :)